<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recModeWindow</class>
 <widget class="QWidget" name="recModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cmdOK">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>98</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>461</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="normalPage">
    <widget class="QPushButton" name="cmdMax">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNormal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Records frames in a single or segmented buffer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="CamSpinBox" name="spinRecLengthFrames">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
     </property>
     <property name="maximum">
      <number>65536</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblRes_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>26</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Record Length (seconds)</string>
     </property>
    </widget>
    <widget class="CamDoubleSpinBox" name="spinRecLengthSeconds">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QDoubleSpinBox { border: 3px outset grey;padding-right: 40px;}  

QDoubleSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QDoubleSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="lblRes_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>106</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Record Length (frames)</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpSegmented">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>331</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Segmented</string>
     </property>
     <widget class="QLabel" name="lblRes_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Segment size (frames):</string>
      </property>
     </widget>
     <widget class="CamSpinBox" name="spinSegmentCount">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65536</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblRes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number of segments</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblNormal_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>Segments buffer into multiple smaller regions</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkSegmentedEnable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {
     width: 40px;
     height: 40px;
}</string>
      </property>
      <property name="text">
       <string>Enable Segmented Memory</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gatedPage">
    <widget class="QLabel" name="lblSegmented_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>481</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Frames are recorded only when the trigger input is active</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Prerecord</string>
     </property>
     <widget class="QLabel" name="lblRes_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Amount of video before each burst to save</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblRes_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prerecord (seconds)</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblRes_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prerecord (frames)</string>
      </property>
     </widget>
     <widget class="CamDoubleSpinBox" name="spinPrerecordSeconds">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>74</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox { border: 3px outset grey;padding-right: 40px;}  

QDoubleSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QDoubleSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="CamSpinBox" name="spinPrerecordFrames">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>154</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;}  

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
      </property>
      <property name="maximum">
       <number>65536</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Record Mode</string>
   </property>
   <widget class="QRadioButton" name="radioNormal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton::indicator {
     width: 40px;
     height: 40px;
}</string>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioGated">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton::indicator {
     width: 40px;
     height: 40px;
}</string>
    </property>
    <property name="text">
     <string>Gated Burst</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSegmented">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton::indicator {
     width: 40px;
     height: 40px;
}</string>
    </property>
    <property name="text">
     <string>Segmented</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="chkDisableRing">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
     width: 40px;
     height: 40px;
}</string>
   </property>
   <property name="text">
    <string>Disable ring buffer</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CamSpinBox</class>
   <extends>QSpinBox</extends>
   <header>camspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>CamDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>camdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
